test/e2e/kind_test.go

```diff
@@ -11,13 +11,14 @@ package e2e
 import (
 	"context"
 	"fmt"
-	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/provisioners"
 	"path/filepath"
 	"strconv"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/provisioners"
+
```

Comment:
This is due to my IDE setting, is no-op and this `kind_test.go` will be removed eventually post refactor